controller/api/models.py

```diff
@@ -672,6 +672,14 @@ def save(self, **kwargs):
                 setattr(self, attr, data)
         except Config.DoesNotExist:
             pass
+
+        for k, v in getattr(self, 'cpu').copy().viewitems():
+            self.cpu[k] = int(v)
```

Comment:
what if someone enters a bad value like `foo` here? I think we should be adding tests for bad values here as well so we're handling both good and bad values properly.